ansible/terraform.py

```diff
@@ -413,12 +413,7 @@ def _backup_ansible(inventory):
     text += '# For emergency use when Consul fails\n'
     text += '[all]\n'
     for hostname, host in sorted(inventory.hosts.items()):
-        text += (
-            '{0} hostname={0} ansible_host={1} '
-        ).format(hostname, host.host_vars['ansible_host']) + (
-            'env={env} stage={stage} data_center={data_center} '+
-            'region={region} dns_entry={dns_entry}\n'
-        ).format(**host.host_vars)
+        text += f"{hostname} " + " ".join("%s=%s" % (k,v) for k,v in host.host_vars.items()) + "\n"
```

Comment:
I would use f-strings instead of `%`.
Maybe also splitting into 2 lines.
```suggestion
vars_text = " ".join(f"{k}={v}" for k,v in host.host_vars.items())
text += f"{hostname} {vars_text}\n"
```